apps/schools/api_views/boundary.py

```diff
@@ -118,7 +118,7 @@ class Admin1s(KLPListAPIView, CacheMixin):
     def get_queryset(self):
         btype = self.request.GET.get('school_type', 'all')
         meetingreport = self.request.GET.get('meetingreport', None)
-        qset = Boundary.objects.all_active()\
+        qset = Boundary.objects.all_active().filter(id__gte=11042)\
             .select_related('boundarycoord__coord', 'hierarchy__name')\
```

Comment:
Hardcoded boundary id for Odisha boundaries. This should ideally be replaced by a `State` level filter for boundaries (probably admin0).